src/rosecommon/include/enums.h

```diff
@@ -17,6 +17,22 @@ enum RidingItem : uint8_t {
     MAX_RIDING_ITEMS
 };
 
+enum ItemSubType : uint16_t {
+    ONEH_SWORD = 211,
+    ONEH_BLUNT, 
+    TWOH_SWORD = 221,
+    SPEAR = 222,
+    TWOH_AXE = 223,
+    BOW = 231,
+    GUN,
+    LAUNCHER,
+    STAFF = 241,
+    WAND,
+    KATAR = 251,
+    DUAL_WEILD,
+    XBOW = 271
```

Comment:
What's the difference between a bow and an xbow?

Reply:
They have a different subtype.
More over, bow = 2h-weap, xbow = 1h-weap